--- app.php	2014-11-30 13:50:44.000000000 -0500
+++ patched.php	2014-11-30 14:20:03.000000000 -0500
@@ -1,5 +1,7 @@
 <?php
-$username = $_POST['username'];  # username field from POST'ed HTML form
-$sql = "SELECT * FROM Person WHERE Username = '$username' ";
-$rs = $db->executeQuery($sql);
+$username = addslashes($_POST['username']);  # username field from POST'ed HTML form
+$sql = "SELECT * FROM Person WHERE Username = ?";
+$query = $db->prepareStatement($sql);
+$query.setString(1, $username);
+$rs = $query->executeQuery();
 ?>
